charts/country-risk/Chart.yaml

```diff
@@ -20,8 +20,8 @@
 apiVersion: v2
 name: country-risk
 type: application
-version: 1.1.1
```

Comment:
Why is there such a huge jump in the umbrella Chart? 
This will make it very confusing for users of your Chart. Could you at least explain this in the `CHANGELOG.md`

Reply:
I increased the chart version to 3.0.3 to be consistent with the subCharts. I think I have been failing to upgrade the version of this whenever there is a change in the charts, and now I have set it to align with them. I can put this explanation in the changelog if it makes it more explanatory.

Reply:
Yes please do that. Jumping major versions and even skipping one seems odd at first sight